pages/quiz.html

```diff
@@ -0,0 +1,153 @@
+              <ul>
+                <li class="lists-item-stacked p2">
+                  Now this is a story all about how, my life got flipped turned upside down
+                </li>
+                <li class="lists-item-stacked  p2">
+                  And I like to take a minute and sit right here
+                </li>
+                <li class="lists-item-stacked  p2">
+                  I'll tell you how I became the prince of a town called Bel-Air
+                </li>
+                <li class="lists-item-stacked danger p2">
+                  And I like to take a minute and sit right here
+                </li>
+                
+              </ul>
+             
+            </div>
+          
+        </div>
+        <div class="wrapper m2">
+            <div class="flex space-around ">
+                <div>Question 3/4</div>
+                <div>
+                    Score:30
+                </div>
+            </div>
+            <div class="heading heading">what do you know about science ?</div>
+            <div class="list-container flex-column-center ">
+              <ul>
+                <li class="lists-item-stacked danger p2">
+                  Now this is a story all about how, my life got flipped turned upside down
+                </li>
+                <li class="lists-item-stacked  p2">
+                  And I like to take a minute and sit right here
+                </li>
+                <li class="lists-item-stacked  p2">
+                  I'll tell you how I became the prince of a town called Bel-Air
+                </li>
+                <li class="lists-item-stacked  p2">
+                  And I like to take a minute and sit right here
+                </li>
+                
+              </ul>
+             
+            </div>
+          
+        </div>
+        <div class="wrapper m2">
+            <div class="flex space-around ">
+                <div>Question 4/4</div>
+                <div>
+                    Score:40
+                </div>
+            </div>
+            <div class="heading heading">what do you know about science ?</div>
+            <div class="list-container flex-column-center ">
+              <ul>
+                <li class="lists-item-stacked p2">
+                  Now this is a story all about how, my life got flipped turned upside down
+                </li>
+                <li class="lists-item-stacked danger p2">
+                  And I like to take a minute and sit right here
+                </li>
+                <li class="lists-item-stacked  p2">
+                  I'll tell you how I became the prince of a town called Bel-Air
+                </li>
+                <li class="lists-item-stacked  p2">
+                  And I like to take a minute and sit right here
+                </li>
+                
+              </ul>
+              <button class="btn quiz-btn m1"><a href="../index.html">Finish</a> </button>
```

Comment:
you should not use a tag inside button tag, you can make you button styles such that it overrides `a` tag's default styles in order to make it look like button